activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```
@@ -152,6 +152,11 @@ instances:
     #
     # kerberos_keytab: <KEYTAB_FILE_PATH>
 
+    ## @param kerberos_cache - string - optional
+    ## Sets KRB5CCNAME environment variable. It should point to a credential cache with a valid TGT.
```

```suggestion
    ## Sets the KRB5CCNAME environment variable. It should point to a credential cache with a valid TGT.
```
Can we update all of these to include `the`?

---

Done